### PR TITLE
Add check for suspended AWS account

### PR DESCRIPTION
**Pull request by SEQUOIIA**
Issue: https://github.com/dfds/cloudplatform/issues/2198

### CODE REVIEW

File: internal/handler/awsmapping.go [hunk 1]
```diff
@@ -66,6 +66,12 @@ func AwsMappingHandler(ctx context.Context) error {
 			return nil
 		default:
 		}
+
+		if resp.Account.Status == "SUSPENDED" {
```
Comment (kralle333) [inline internal/handler/awsmapping.go]: Can you extract "SUSPENDED" to a constant

File: internal/handler/awsmapping.go [hunk 2]
```diff
@@ -66,6 +66,12 @@ func AwsMappingHandler(ctx context.Context) error {
...
+			util.Logger.Debug(fmt.Sprintf("Suspended account detected with missing Capability access - %s (%s), skipping account", *resp.Account.Name, *resp.Account.Id), zap.String("jobName", AwsMappingName))
```
Comment (kralle333) [inline internal/handler/awsmapping.go]: Can you make it a warning instead?

**Review by kralle333 — CHANGES_REQUESTED** [review]
Nice, just small comments

File: internal/handler/awsmapping.go (hunk 1 above)
Comment (SEQUOIIA) [reply internal/handler/awsmapping.go]: https://github.com/dfds/aad-aws-sync/pull/31/commits/f87996060998563ed5555f6b12a56037a444ede3

File: internal/handler/awsmapping.go (hunk 2 above)
Comment (SEQUOIIA) [reply internal/handler/awsmapping.go]: https://github.com/dfds/aad-aws-sync/pull/31/commits/f87996060998563ed5555f6b12a56037a444ede3

**Review by kralle333 — APPROVED** [review]
SWEET

**Pull request merged**